RFS/src/testFixtures/java/org/opensearch/migrations/bulkload/framework/SearchClusterContainer.java

```diff
@@ -68,6 +69,11 @@ private enum INITIALIZATION_FLAVOR {
                 .build()),
         OPENSEARCH(
             new ImmutableMap.Builder<String, String>().putAll(BASE.getEnvVariables())
+                .put("plugins.security.disabled", "true")
+                .put("OPENSEARCH_INITIAL_ADMIN_PASSWORD", "SecurityIsDisabled123$%^")
+                .build()),
+        OPENSEARCH_2_19(
+        new ImmutableMap.Builder<String, String>().putAll(BASE.getEnvVariables())
```

Comment:
Can we add the new entry for OPENSEARCH rather than create a new category?

Reply:
No, if it's present in earlier opensearch versions it prevents the cluster from initializing